app/src/main/java/com/kamron/pogoiv/DevMethods.java

```diff
@@ -0,0 +1,67 @@
+package com.kamron.pogoiv;
+
+import java.util.ArrayList;
+
+/**
+ * Created by Johan on 2016-08-27.
+ *
+ * A class for methods which is only used during development
+ */
+public class DevMethods {
+
+
+    /**
+     * Method used to generate the <item> </item> list for candy cost of evolution
+     * Since pokemons follow the following pattern (with a few exceptions):
+     * Pokemon with 3 evolutions will cost
+     * 25,100,-
+     * 2 evolutions
+     * 50, -
+     * Exceptions: Eevee, weedle, caterpie, pidgey, magicarp and rattatta
+     *
+     * To run this method, you need to make the pokemon field evolutionCandyCost not final and uncomment the method
+     */
+    private void printOutEvolutionCandyCosts(PokeInfoCalculator pokeCalculator){
+        //to create the evolutions
+        for (Pokemon poke : pokeCalculator.pokedex){
+            ArrayList<Pokemon> evoLine = pokeCalculator.getEvolutionLine(poke);
+            int numberInEvoLine =1;
+            for (int i = 0; i<evoLine.size(); i++){
+                System.out.println("poke:" + poke.name + " evoLine size: " + evoLine.size());
+                if (poke.name.equals(evoLine.get(i).name)){
+                    numberInEvoLine = i;
+                }
+            }
```

Comment:
Please explain right here (before the commented out code) *why* this is commented out. You need something like:

> This is commented out because the evolution cost variable is final in the running build, but the method rewrites the cost, so it gives a compile warning if it's not commented out.

No less information will do. Rationale: ensure people who clean up dead code know what to do here.